CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 9.16
```

Comment:
I hadn't noticed that you created this file.
Is the plan to use this file to document changes instead of the releases?

Reply:
sounds good. We can move other recent notes to it later.